docs/requirements/stakeholder/index.rst

```diff
@@ -264,9 +264,71 @@ Dependability
    * ECC Memory
    * Software Lockstep
    * Power management integrated circuit (PMIC), external watchdog and voltage monitoring
-   * Safe switch from engineering for field mode and back
+   * Safe switch from engineering to field mode and back
 
-   Note: This is part of 0.5 release and therefore can only support ASIL_B. Goal is ASIL_D.
+
+.. stkh_req:: Safe SW-platform state
+   :id: stkh_req__dependability__safe_state
+   :reqtype: Functional
+   :security: NO
+   :safety: ASIL_B
+   :rationale: A safe state definition is expected by every safety standard.
+   :status: valid
+   :tags: safety_mechanism
+
+   The SW-platform shall react in the following way on errors:
+
+   - in case of an unrecoverable error (e.g. access violations), not reporting healty status to the external health monitor
```

Comment:
I am not fully getting this sentence. Is the intention to say:

a)
```
   - in case of an unrecoverable error (e.g. access violations), report the non-healthy status to the external health monitor
```

b)
```suggestion
   - in case of an unrecoverable error (e.g. access violations), which cannot be reported to the external health monitor
```